src/test/regress/pg_regress_multi.pl

```diff
@@ -550,13 +550,20 @@ sub generate_hba
 {
    push(@pgOptions, "citus.worker_min_messages='warning'");
    push(@pgOptions, "citus.log_distributed_deadlock_detection=on");
-   push(@pgOptions, "citus.distributed_deadlock_detection_factor=-1");
    push(@pgOptions, "citus.shard_count=4");
    push(@pgOptions, "citus.metadata_sync_interval=1000");
    push(@pgOptions, "citus.metadata_sync_retry_interval=100");
    push(@pgOptions, "client_min_messages='warning'"); # pg12 introduced notice showing during isolation tests
    push(@pgOptions, "citus.running_under_isolation_test=true");
 
+   # Disable all features of the maintenance daemon. Otherwise queries might
+   # randomly show temporarily as "waiting..." because they are waiting for the
+   # maintenance daemon.
+   push(@pgOptions, "citus.distributed_deadlock_detection_factor=-1");
```

Comment:
I always wanted to have a global GUC to disable all maintenance daemon, now I have a use case :p